pkg/docker/images/drive-harddisk-symbolic.svg

```diff
@@ -0,0 +1,34 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
@andreasn What about using http://fontawesome.io/icon/hdd-o/ instead of our own image?